microsoft-edge/webview2/release-notes.md

```diff
@@ -93,6 +104,132 @@ SDKs/packages are Prerelease or Release
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.isgeneralautofillenabled?view=webview2-dotnet-1.0.1829-prerelease&preserve-view=true)
+* [CoreWebView2Profile.IsPasswordAutosaveEnabled Property](/dotnet/api/microsoft.web.webview2.core.corewebview2profile.ispasswordautosaveenabled?view=webview2-dotnet-1.0.1829-prerelease&preserve-view=true)
+
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Profile Class](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
+   * [CoreWebView2Profile.IsGeneralAutofillEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile#isgeneralautofillenabled?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
+   * [CoreWebView2Profile.IsPasswordAutosaveEnabled Property](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2profile#ispasswordautosaveenabled?view=webview2-winrt-1.0.1829-prerelease&preserve-view=true)
+
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Profile6](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true)
+   * [ICoreWebView2Profile6::get_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#get_ispasswordautosaveenabled)
+   * [ICoreWebView2Profile6::put_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_ispasswordautosaveenabled)
+   * [ICoreWebView2Profile6::put_IsPasswordAutosaveEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_ispasswordautosaveenabled)
+   * [ICoreWebView2Profile6::get_IsGeneralAutofillEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#get_isgeneralautofillenabled)
+   * [ICoreWebView2Profile6::put_IsGeneralAutofillEnabled](/microsoft-edge/webview2/reference/win32/icorewebview2profile6?view=webview2-1.0.1829-prerelease&preserve-view=true#put_isgeneralautofillenabled)
+---
+
+<!-- ------------------------------ -->
+###### Bug fixes
+
+* Disable the Chrome Web Store info banner that displays the option to allow extensions installation. ([Issue #3312](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3312))
+* [TODO: Add description here] ([Issue #3300](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3300))
+* [TODO: Add description here] ([Issue #3375](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3375))
+* Fixed a bug in PostSharedBufferToScript ([Issue #3360] (https://github.com/MicrosoftEdge/WebView2Feedback/issues/3360)). Runtime only.

```

Comment:
```suggestion
* Fixed a bug in PostSharedBufferToScript ([Issue #3360](https://github.com/MicrosoftEdge/WebView2Feedback/issues/3360)). Runtime only.
```

Reply:
checking latest Files Changed tab...
present view/listing is Outdated.

Reply:
the problem remains.  fixing...